vaadin-card-flow-parent/vaadin-card-flow-integration-tests/src/test/java/com/vaadin/flow/component/card/tests/CardIT.java

```diff
@@ -42,5 +44,30 @@ public void rendersCardComponent() {
 
         Assert.assertTrue(hasShadowRoot);
         Assert.assertEquals("vaadin-card", componentName);
+        Assert.assertNotNull(card.getTitle());
+        Assert.assertNotNull(card.getSubtitle());
+        Assert.assertNotNull(card.getMedia());
+        Assert.assertNotNull(card.getHeaderPrefix());
+        Assert.assertNotNull(card.getHeaderSuffix());
+        Assert.assertFalse(card.getFooterContents().isEmpty());
+        Assert.assertFalse(card.getContents().isEmpty());
+    }
+
+    @Test
+    public void setTitleComponent_setTitleString_titleComponentIsRemoved() {
+        clickElementWithJs("set-title-component");
+        var titleComponent = card.getTitle();
+        clickElementWithJs("set-string-title");
+        Assert.assertNotEquals(titleComponent, card.getTitle());
+        Assert.assertEquals("String title", card.getStringTitle());
+    }
+
+    @Test
+    public void setStringTitle_setTitleComponent_stringTitleIsRemoved() {
+        clickElementWithJs("set-string-title");
+        var stringTitleComponent = card.getTitle();
+        clickElementWithJs("set-title-component");
+        Assert.assertNotEquals(stringTitleComponent, card.getTitle());
+        Assert.assertNull(card.getStringTitle());
```

Comment:
Do we need these? It seems that this is already covered by unit tests.

Reply:
Removed the ITs.